pom.xml

```diff
@@ -205,6 +206,16 @@
       <artifactId>google-api-services-storage</artifactId>
       <version>${google-storage-api.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.http-client</groupId>
```

Comment:
Beam provides a JsonFactory in org.apache.beam.sdk.util.Transport that can be used for serializing these objects to Json. These pom.xml changes should not be needed in that case.

Reply:
I tried to remove `com.google.http-client` dependency.
But this error has occurred to:

```
[INFO] --- maven-dependency-plugin:3.0.1:analyze-only (analyze-dependencies) @ google-cloud-teleport-java ---
[WARNING] Used undeclared dependencies found:
[WARNING]    com.google.http-client:google-http-client:jar:1.27.0:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:18 min
[INFO] Finished at: 2019-06-09T01:32:03+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.0.1:analyze-only (analyze-dependencies) on project google-cloud-teleport-java: Dependency problems found -> [Help 1]
```